mevislab.github.io/content/tutorials/basicmechanisms/coordinatesystems/coordinatesystems.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Example 1.1: MeVisLab Coordinate Systems"
+date: 2022-06-15T08:58:44+02:00
+status: "OK"
+draft: false
+tags: ["Beginner", "Tutorial",  "Data Import", "DICOM", "Coordinate Systems"]
+menu: 
+  main:
+    identifier: "coordinatesystems"
+    title: "The different coordinate systems in MeVisLab: World-, Voxel- and Device coordinates."
+    weight: 365
+    parent: "data_import"
+---
+
+# Example 1.1: MeVisLab Coordinate Systems
+Three coordinate systems exist next to each other:
+* World coordinates
+* Voxel coordinates
+* Device coordinates
+
+World coordinate systems in MeVisLab are always [right handed](https://en.wikipedia.org/wiki/Right-hand_rule).
+
+The blue rectangle shows the same region in the three coordinate systems.
+
+![Coordinate Systems in MeVisLab](/images/tutorials/basicmechanics/GSExampleImageProcessing10b.png "Coordinate Systems in MeVisLab")
+
+## World coordinates
+World coordinates are:
+* Global: Combine several objects in a view
+* Absolute: Measure distances and angles
+* Isotropic: All directions are equivalent
+* Orthogonal: Coordinate axes are orthogonal to each other
+
+The origin of the world coordinate system can be anywhere and is not clearly defined. Coordinates of the other systems are always relative to the origin of the world coordinates.
```

Comment:
"Coordinates of the other systems are always relative to the origin of the world coordinates." - no, they are not. They have their own origin, defined by the upper left pixel / voxel.
Maybe you confuse the coordinate systems with the transformation matrices?